docs/source/complex_numbers.rst

```diff
@@ -0,0 +1,174 @@
+.. _complex_numbers-doc:
+
+Complex Numbers
+===============
+
+Complex numbers are numbers that can be expressed in the form :math:`a + bj`, where a and b are real numbers,
+and *j* is a solution of the equation :math:`x^2 = −1`. Complex numbers frequently occur in mathematics and
+engineering, especially in signal processing. Tensors of complex dtypes provide a more natural user experience
+for users and libraries (eg. TorchAudio) that previously worked around the lack of complex tensors by using
+float tensors with shape :math:`(..., 2)` where the last dimension contained the real and imaginary values.
+
+Operations on complex tensors (eg :func:`torch.mv`, :func:`torch.matmul`) are likely to be faster and more
+memory efficient than operations on float tensors mimicking them. Operations involving complex numbers in
+PyTorch are optimized to use vectorized assembly instructions and specialized kernels (e.g. LAPACK, CuBlas).
+Thus using functions for complex tensors will provide performance benefits as opposed to users defining
+their own functions.
+
+.. note::
+     Spectral Ops currently don't use complex tensors but the API would be soon updated to use complex tensors.
```

Comment:
nit: I think there's a tense disagreement; if you say "currently don't" then you should say "API will soon be updated"

Reply:
nit: Spectral operations (don't abbreviate, don't capitalize)

Reply:
nit: good to mention an e.g., list of what spectral operations are